src/olympia/git/admin.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib import admin
+from django.db.models import Prefetch
+from django.utils.html import format_html
+
+from olympia.addons.models import Addon
+from olympia.amo.urlresolvers import reverse
+
+from .models import GitExtractionEntry
+
+
+@admin.register(GitExtractionEntry)
+class GitExtractionEntryAdmin(admin.ModelAdmin):
+    actions = None
+    view_on_site = False
+
+    list_display = ('id', 'formatted_addon', 'in_progress', 'created')
```

Comment:
Maybe add `modified` ? I believe it should be updated when we set `in_progress=True`, so that could be useful info maybe ?